modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-user-provided-vnet-subnet"]
+= Configuring a private storage endpoint on Azure with user-provided VNet and subnet names
+
+Use the following procedure to configure a private storage endpoint on Azure with user-provided VNet and subnet names.
+
+.Prerequisites 
+
+* You have configured the Image Registry Operator for use with Azure.
+
+.Procedure 
+
+. Enter the following command to obtain the VNet and subnet names that are used in the cluster:
++
+[source,terminal]
+----
+$ oc get machinesets -nopenshift-machine-api -ojson|jq '.items[1].spec.template.spec.providerSpec.value | (.vnet, .subnet)'
+----
+
+. Edit the Image Registry Operator `config/instance` object and configure the private endpoint using your VNet and subnet names:
++
+[source,terminal]
+----
+$ oc edit configs.imageregistry/cluster
+----
++
+[source,terminal]
+----
+# ...
+spec:
+  [...]
+   storage:
+      azure:
+        [...]
+        networkAccess:
+          type: Internal
+          internal:
+            subnetName: <subnet_name>
+            vnetName: <vnet_name>
+# ...
+----
+
+. Wait until the Image Registry Operator configures the private endpoint. Enter the following command to return the endpoint name:
++
+[source,terminal]
+----
+$ oc get configs.imageregistry/cluster -o=jsonpath="{.spec.storage.azure.privateEndpointName}" -w
+----
+
+. Enter the following command to enable the default route and disable `redirect` on the Image Operator configuration:
++
+[source,terminal]
+----
+$ oc patch configs.imageregistry cluster --type=merge -p '{"spec":{"defaultRoute": true, "disableRedirect": true}}'
+----
+
+.Verification
```

Comment:
What do we usually want to achieve in the verification section?
Showing that pulls work does not prove that the storage account is private. If we want to prove that the previous configuration worked, we might want to show that pulling via default route _without disabling redirect_ won't work - this is the way to prove that the private endpoint configuration worked.
I'm not sure if it makes sense to do this here tho... WDYT?

Reply:
I like it! left some comments there.